### PR TITLE
My Recipes UI

### PR DESCRIPTION
**Pull request by seishinleblanc**
### Description of the changes included in this pull request

Created basic UI of MyRecipes page. Added a My Recipes Search Bar component and a My Recipes Card Container component rather than reusing the existing components since I expect those may behave a bit differently than the Search Bar and Recipes Card components in the following ways:

-The My Recipes Search Bar will need to search the user favorites rather than all recipes.

-The My Recipes Card Container component will need to map from the user's favorites rather than all recipes.

### Screenshots

Screenshot 2024-04-17 at 10 12 04 AM

### CODE REVIEW

**Review by bduran04 — DISMISSED** [review]
Looks good! Nice job on using mock data

**Review by coffee2theorems — APPROVED** [review]

**Review by coffee2theorems — APPROVED** [review]

**Review by bethanyann — APPROVED** [review]
approving again after merge

**Comment by Dissurender** [comment]
I'm going to go ahead and merge this in to avoid some blockers on the Recipe page. :shamrock:

**Pull request merged**
